src/ports/postgres/modules/utilities/validate_args.py_in

```diff
@@ -458,6 +458,22 @@ def scalar_col_has_no_null(tbl, col):
 # -------------------------------------------------------------------------
 
 
+def array_col_dimension(tbl, col):
+    """
+    What is the dimension of this array column
+    """
+    if tbl is None or tbl.lower() == 'null':
+        plpy.error('Input error: Table name (NULL) is invalid')
+    if col is None or col.lower() == 'null':
```

Comment:
IMO if we shouldn't be checking for the string 'NULL'. There are multiple strings that are invalid table/column names - why make 'null' an exception?